index.html

```diff
@@ -159,141 +156,401 @@ <h2>
+        <h3>
+          `PointerLockOptions` dictionary
+        </h3>
+        <pre class="idl">
+          dictionary PointerLockOptions {
+            boolean unadjustedMovement = false;
+          };
+        </pre>
+        <dl>
+          <dt>
+            <dfn>PointerLockOptions</dfn> dictionary
+          </dt>
+          <dd>
+            <p>
+              The options dictionary to customize how the pointer behaves in
+              the locked mode.
+            </p>
+          </dd>
+          <dt>
+            <dfn>unadjustedMovement</dfn> member
+          </dt>
+          <dd>
+            <p>
+              If this value is set to `true`, then the pointer movements will
+              not be affected by the underlying platform modications such as
+              mouse accelaration.
+            </p>
+          </dd>
+        </dl>
+      </section>
+      <section>
+        <h3>
+          `pointerlockchange` and `pointerlockerror` Events
+        </h3>
+        <p>
+          Two events are used to communicate pointer lock state change or an
+          error in changing state. They are named <dfn>pointerlockchange</dfn>
+          and <dfn>pointerlockerror</dfn>. If pointer lock is entered or exited
+          for any reason a {{Document/pointerlockchange}} event must be sent.
+        </p>
+        <p>
+          The [=user agents=] must deliver these events by <a data-lt=
+          "queue a task">queuing a task</a> to [=fire an event=] of the
+          appropriate name with its `bubbles` attribute [[DOM]] set to false to
+          the pointer lock [=Event/target=] element's [=Node/node document=].
+        </p>
+        <p class="note">
+          Magnification software increases the size of content on the screen.
+          It uses the mouse to move the magnified point of focus around. When a
+          pointer lock is initiated, the magnification software needs to switch
+          to using the keyboard to move the magnified point of focus around
+          instead. When a {{pointerlockchange}} event is fired, web browsers
+          therefore need to make sure the event is communicated to assistive
+          technologies like screen magnifiers.
+        </p>
+      </section>
+      <section>
+        <h3>
+          <dfn>Exit Pointer Lock</dfn>
+        </h3>
+        <p>
+          The process of exiting pointer lock, given an element, is as follows:
+        </p>
+        <ol>
+          <li>The system mouse cursor must be displayed again and positioned at
+          [=cursor position=]
+          </li>
+          <li>[=Queue an element task=], given this element, on the [=user
+          interaction task source=] to do the following:
+            <ol>
+              <li>[=Fire an event=] named {{pointerlockchange}} at the given
+              element's [=Node/node document=]
+              </li>
+            </ol>
+          </li>
+          <li>Exit the [=pointer lock state=] by setting [=pointer-lock
```

Comment:
I wish we had these as internal slots that are set... would be much nicer but this is just a nit.